types/wlr_output_management_v1.c

```diff
@@ -680,6 +680,7 @@ static void head_send_state(struct wlr_output_head_v1 *head,
 
 	if (state & HEAD_STATE_ENABLED) {
 		zwlr_output_head_v1_send_enabled(head_resource, head->state.enabled);
+		state = ~(state & 0);
```

Comment:
just wonder if setting `state = HEAD_STATE_ALL` improves readbility?

Reply:
Definitely. Didn't notice this variable. I'll change it. Thanks!